Functions-request-config.js

```diff
@@ -43,8 +43,9 @@ const requestConfig = {
   expectedReturnType: ReturnType.uint256,
   // Redundant URLs which point to encrypted off-chain secrets
   secretsURLs: [],
+  // Default offchain secrets object used by the `functions-build-offchain-secrets` command
+  globalSecrets: {},
```

Comment:
Can we remove "secrets" from this file, so as not to confuse users?

Reply:
secrets is now user for on-chain secrets.  I thought it actually made more sense to do it this way and let to less confusion overall, but let me know if you feel otherwise.

Reply:
I could call it `globalOffchainSecrets` and `perNodeOffchainSecrets` to be even more explicit.